s3repo/model.py

```diff
@@ -73,7 +95,7 @@ def __init__(self, s3_settings):
         # needs to be updated. All actions with "unsync_repos" must
         # be done under the "sync_lock".
         self.sync_lock = Lock()
-        self.unsync_repos = set()
+        self.unsync_repos: Set = set()
```

Comment:
set of what?

Reply:
Now - `str` , in the near future `RepoInfo`.